### PR TITLE
CORS-3185: update default release image

### PR DESCRIPTION
**Pull request by rna-afk**
Updating the release image to the latest stable version 4.16.

### CODE REVIEW

**Review by r4f4 — COMMENTED** [review]
/lgtm

File: cmd/openshift-install/testdata/agent/image/manifests/default_manifests.txt [hunk 1]
```diff
@@ -101,7 +101,7 @@ metadata:
   name: openshift-was not built correctly
   namespace: cluster0
 spec:
-  releaseImage: registry.ci.openshift.org/origin/release:4.14
+  releaseImage: registry.ci.openshift.org/origin/release:4.15
```
Comment (patrickdillon) [inline cmd/openshift-install/testdata/agent/image/manifests/default_manifests.txt:104]: unfortunately we missed doing this in 4.15 before branching

```suggestion
  releaseImage: registry.ci.openshift.org/origin/release:4.16
```

File: pkg/asset/releaseimage/default.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 
 var (
 	// defaultReleaseImageOriginal is the value served when defaultReleaseImagePadded is unmodified.
-	defaultReleaseImageOriginal = "registry.ci.openshift.org/origin/release:4.14"
+	defaultReleaseImageOriginal = "registry.ci.openshift.org/origin/release:4.15"
```
Comment (patrickdillon) [inline pkg/asset/releaseimage/default.go:23]:
```suggestion
	defaultReleaseImageOriginal = "registry.ci.openshift.org/origin/release:4.16"
```

**Review by barbacbd — COMMENTED** [review]
/lgtm

**Comment by barbacbd** [comment]
New PR opened https://github.com/openshift/installer/pull/7936 for 4.15

**Comment by andfasano** [comment]
integration-tests should be updated as well

**Comment by patrickdillon** [comment]
> integration-tests should be updated as well

@andfasano can you help @rna-afk update the tests so we can get this closed out? Is it possible to update the tests so this value, which is expected to change, does not require bumping the tests?

**Comment by patrickdillon** [comment]
#7992 should merge soon. so we can drop all the test updates

**Comment by patrickdillon** [comment]
instead of rebasing, you can just drop all the tests updates, since andrea fixed them to be dynamic

**Review by barbacbd — COMMENTED** [review]
/lgtm

**Review by r4f4 — COMMENTED** [review]
/approve

**Pull request merged**